src/styles/main.scss

```diff
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import "./untils/reset";
```

Comment:
where is other scss files? use `git add .` => `git commit -m 'solution'` instead of `git commit -am 'solution'`

Reply:
I have it)
[files](https://github.com/Viktormaz/layout_calendar/tree/develop/src/styles)

Reply:
they should be also in the pull request